pycobertura/filesystem.py

```diff
@@ -68,3 +71,52 @@ def open(self, filename):
             f.close()
         except KeyError:
             raise self.FileNotFound(filename)
+
+
+class GitFileSystem(FileSystem):
+    def __init__(self, repo_folder, ref):
+        self.repository = repo_folder
+        self.ref = ref
+        self.repository_root = self._get_root_path(repo_folder)
+        self.prefix = self.repository.replace(self.repository_root, "").lstrip("/")
```

Comment:
Can you elaborate on how `prefix` is used in the context of `GitFileSystem`? It's not immediately obvious by reading the code.